documentation/src/docs/admin-guide/lts-sizing.md

```diff
@@ -106,4 +110,47 @@ the thread pool size to 2-4 threads.
 - _Maximum size of single chunk in LTS: In production, it may be required to revisit the value of `writer.rollover.size.bytes.max`.
 Limiting the size of chunks in LTS poses a trade-off: if the value is low, the system will achieve fine retention granularity 
 (good) but extra metadata overhead (not so good). On the contrary, a high value in this parameter induces coarse retention 
-granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios. 
\ No newline at end of file
+granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios.
+
+## Right-Sizing Tier-1 Storage (Bookkeeper)
+
+Pravega makes use of tier-1 storage to store durably and with low latency events until they are moved to long-term storage. 
```

Comment:
'to store durably and with low latency events' -> 'to store events durably and with low latency' OR 'for durable, low-latency storage of events'

Given eventstreams & bytestreams, perhaps wording it like this helps:
'Pravega uses tier-1 storage for durable, low-latency persistence until data can be moved to long-term storage.'

Reply:
Fixed.